pkg/manager/manager.go

```diff
@@ -242,6 +244,13 @@ type Options struct {
 	// It is used to set webhook.Server.CertDir if WebhookServer is not set.
 	CertDir string
 
+	// TLSMinVersion is the minimum version of TLS supported by the webhook server.
+	// Accepts "", "1.0", "1.1", "1.2" and "1.3" only ("" is equivalent to "1.0" for backwards compatibility).
+	TLSMinVersion string
+
+	// TLSOpts is used to allow configuring the TLS config used for the webhook server.
+	TLSOpts []func(*tls.Config)
```

Comment:
Why add both the `TLSOpts` and the `TLSMinVersion`?

Reply:
Good point. I basically just added both because we have both on the Webhook Server.

But of course TLSOpts can be used to set the min version as well. For me TLSOpts would be enough.

Nevertheless, wondering if we want to have TLSMinVersion on `ControllerWebhook` (because there TLSOpts cannot be added).

Reply:
We should probably only keep a single field, in this case sounds like TLSOpts would be the way to go, given that it allows more customization?